demo-app/src/main/java/io/opentelemetry/android/demo/OtelDemoApplication.kt

```diff
@@ -62,6 +61,26 @@ class OtelDemoApplication : Application() {
         } catch (e: Exception) {
             Log.e(TAG, "Oh no!", e)
         }
+
+        // This is needed to get R8 missing rules warnings.
+        initializeOtelWithGrpc()
+    }
+
+    // This is not used but it's needed to verify that our consumer proguard rules cover this use case.
+    private fun initializeOtelWithGrpc() {
+        val builder = OpenTelemetryRum.builder(this)
+            .addSpanExporterCustomizer {
+                OtlpGrpcSpanExporter.builder().build()
+            }
+            .addLogRecordExporterCustomizer {
+                OtlpGrpcLogRecordExporter.builder().build()
+            }
+
+        // This is an overly-cautious measure to prevent R8 from discarding away the whole method
+        // in case it identifies that it's actually not doing anything meaningful.
+        if (System.currentTimeMillis() < 0) {
+            print(builder)
+        }
```

Comment:
I guess we can remove this now?

Reply:
@LikeTheSalad I'm going to merge this because it looks good and this change isn't in the critical path, but hopefully you can revisit/respond to it. Thanks!